Source/SwipeToRevealView.swift

```diff
@@ -1,8 +1,17 @@
 import UIKit
 import SnapKit
 
+@objc public protocol SwipeToRevealViewDelegate {
+  @objc optional func swipeToRevealView(view: SwipeToRevealView, didClose animated: Bool)
+  @objc optional func swipeToRevealView(view: SwipeToRevealView, didRevealRight animated: Bool)
+  @objc optional func swipeToRevealView(view: SwipeToRevealView, didPan pan: UIPanGestureRecognizer)
+}
```

Comment:
Sounds good! I just made the protocol `objc` so the functions could be optional.